sql.txt

```diff
@@ -1,14 +1,29 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country ASC LIMIT 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers WHERE ContactName LIKE "a%";
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) 
```

Comment:
This looks great! Seems like you have a firm understanding of SQL. Thought you would like to know this, since we are adding values for all the columns of the Products Table, you actually don't need to specify each column name for the INSERT INTO clause (just need to ensure you keep the order of the values the same as the order of the columns). So, you could do this:

```
INSERT INTO Products
VALUES (NULL, "Earl Grey tea", 20, 1, "115 g", 7.99), 
(NULL, "Ceylon tea", 20, 1, "125 g", 12.50), 
(NULL, "Indian tea", 20, 1, "250 g", 18.50);
```